### PR TITLE
Use owncloudci/client as the base image

### PR DESCRIPTION
**Pull request by saw-jan**
Similar to 003d36b79d9aab7e4d79ea1bdfb9d83541d738c1
Need to run desktop-client v5.0.0

Fixes https://github.com/owncloud/client/issues/10784 and https://github.com/owncloud-ci/squish/issues/25

### CODE REVIEW

**Comment by TheOneRing** [comment]
@fmoc please coordinate

**Comment by saw-jan** [comment]
I would like to try it first with squish tests pipeline.

**Comment by saw-jan** [comment]
> I would like to try it first with squish tests pipeline.
> 
> Let's see the build: https://drone.owncloud.com/owncloud/client/15181/6/4

Passing build: https://drone.owncloud.com/owncloud/client/15181/6/7

File: fedora/Dockerfile.amd64 [hunk 1]
```diff
@@ -1,5 +1,5 @@
-ARG BASE=fedora
-ARG BASETAG=36
+ARG BASE=owncloudci/client
+ARG BASETAG=fedora-36-amd64
```
Comment (TheOneRing) [inline fedora/Dockerfile.amd64:2]: 38

**Review by fmoc — APPROVED** [review]

**Comment by saw-jan** [comment]
@fmoc @TheOneRing Could you guys merge it if it is okey to, because I don't have permission to merge here.

**Pull request merged**